tools/codegen.mk

```diff
@@ -9,7 +9,9 @@ endif
 # Generate code
 code-generate: controller-gen
```

Comment:
Updated these commands:

- The first command was not working with more recent versions of controller-gen
- The second command was necessary to (re-)generate the `*.deepcopy.go` files